packages/react-dom-bindings/src/client/ReactDOMFloatClient.js

```diff
@@ -174,14 +179,13 @@ function preconnectAs(
 
       const preconnectProps = {rel, crossOrigin, href};
       if (null === ownerDocument.querySelector(key)) {
-        const preloadInstance = createElement(
+        const preloadInstance = createHTMLElement(
```

Comment:
This function is just a bunch of checks on whether or not it's a specific type or props has an esoteric `is` flag. Neither of which apply here.

This is a classic case of the over-branching that the old style of DOM bindings caused and I tried to fix with Fizz.

Just use `ownerDocument.createElement` in Float.